modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
what if `'beta.o' %in% vname`is false? Wouldn't an error occur later in the absence of the object `dat`? Wouldn't the absence of `dat` imply something was wrong with the meta-analysis?